base/common/src/test/java/com/netscape/cms/servlet/test/ConfigurationTest.java

```diff
@@ -539,66 +522,4 @@ private static void constructTKSData(String host, String port, String pin, Strin
         cert5.setToken(CryptoUtil.INTERNAL_TOKEN_FULL_NAME);
         systemCerts.add(cert5);
     }
-
-    private static String generateCRMFRequest(CryptoToken token, String keysize, String subjectdn, boolean dualkey)
```

Comment:
Could you restore this method for now? I'd like to convert it into a CLI. You can change it to a `public` method to get rid of the warning.

Reply:
Done